app/javascript/components/pages/key-performance-indicators/container.jsx

```diff
@@ -0,0 +1,83 @@
+import { fromJS } from "immutable";
+import PropTypes from "prop-types";
+import React, { useEffect } from "react";
+import { withRouter } from "react-router-dom";
+import { connect, batch } from "react-redux";
+import { PageContainer } from "components/page-container";
+import { Grid, Box } from "@material-ui/core";
+import { OptionsBox, DashboardTable } from "components/dashboard";
+import { useI18n } from "components/i18n";
+import makeStyles from "@material-ui/styles/makeStyles";
+import styles from "./styles.css";
+import * as actions from "./action-creators";
+import * as selectors from "./selectors";
+
+function KeyPerformanceIndicators({ fetchNumberOfCases, numberOfCases }) {
+  const i18n = useI18n();
+  const css = makeStyles(styles)();
+  useEffect(() => {
+    batch(() => {
+      fetchNumberOfCases();
+    });
+  }, []);
+
+  return (
+    <div>
+      <PageContainer>
```

Comment:
I believe next time you pull there should be a PageHeading and a PageContent component you can use instead for the heading.

```
<PageContainer>
  <PageHeading title="Hello" />
  <PageContent>...</PageContent>
</PageContainer>
```

Anything passed as children to PageHeading will float to the right of the title. Also the path to those components was changed from `components/page-container` to `components/page`